### ISSUE TITLE
Improve color contrast

### ISSUE THREAD

**Issue by dneufeldcu** [cedardevs/onestop#269]
Recommendations:

- [x] Text link to API docs use #277cb2
- [ ] ~~Action buttons pop e.g. 'Show Matching Files' should pop - #277cb2 ?~~

**Comment by ajakz** [comment]
Changed link to API docs to #277cb2; Show Matching Files button will become its own prominent tab on the collection summary changes that should be rolled into the release.
